index.html

```diff
@@ -11,7 +11,7 @@
 </head>
 
 <body>
-  <div class="master-container">
+  <div class="index-master-container">
```

Comment:
I think you are repeating the same class all over again with each page and there's no need for that. You can have a common `master-container` class with all the common attributes and add the classes with the overwrite of what you need. For example:

```scss
.master-container{
  display: flex;
  height: 768px;
  width: 100%;
  max-width: 1366px;
  justify-content: center;
  align-items: center;
}

.index-container {
  background-color: $white;
}
```

and in the HTML:

```html
<div class="master-container index-container">
...
```